Sources/Animators/Common/Navigator.swift

```diff
@@ -26,6 +26,7 @@ public class Navigator: NSObject {
 
     animator = AnimatorFactory.makeAnimator(transitionAnimationType: transitionAnimationType, transitionDuration: transitionDuration)
 
+    #if os(iOS)
     // If interactiveGestureType has been set
     if let interactiveGestureType = interactiveGestureType {
       // If configured as `.Default` then use the default interactive gesture type from the Animator
```

Comment:
This could be extracted into another initializer only for iOS. 
It may duplicate code but I think its better than a nested OS condition.